src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/shared/resources/options/fetch/fetchOptionsSagas.test.ts

```diff
@@ -1,10 +1,11 @@
 import { expectSaga, testSaga } from 'redux-saga-test-plan';
 import { select } from 'redux-saga/effects';
 import FormDataActions from 'src/features/form/data/formDataActions';
-import { checkIfOptionsShouldRefetchSaga, fetchSpecificOptionSaga, formDataSelector, instanceIdSelector, optionsSelector, userLanguageSelector, watchCheckIfOptionsShouldRefetchSaga } from 'src/shared/resources/options/fetch/fetchOptionsSagas';
+import { checkIfOptionsShouldRefetchSaga, fetchOptionsSaga, fetchSpecificOptionSaga, formDataSelector, formLayoutSelector, instanceIdSelector, optionsSelector, userLanguageSelector, watchCheckIfOptionsShouldRefetchSaga } from 'src/shared/resources/options/fetch/fetchOptionsSagas';
 import { IOptions, IRuntimeState } from 'src/types';
 import * as networking from 'altinn-shared/utils/networking';
 import { IInstance } from 'altinn-shared/types';
+import { ILayouts, ISelectionComponentProps } from 'src/features/form/layout';
```

Comment:
Change to type imports? :) 

```suggestion
import type { IOptions, IRuntimeState } from 'src/types';
import * as networking from 'altinn-shared/utils/networking';
import type { IInstance } from 'altinn-shared/types';
import type { ILayouts, ISelectionComponentProps } from 'src/features/form/layout';
```